infrastructure/transform/main.tf

```diff
@@ -65,6 +65,45 @@ resource "azurerm_data_factory" "adf" {
   }
 }
 
+
+resource "azurerm_data_factory_pipeline" "pipeline" {
+  name            = "databricks-pipeline-${var.naming_suffix}"
+  data_factory_id = azurerm_data_factory.adf.id
+  activities_json = <<JSON
+[
+  {
+    "name": "DatabricksPythonActivity",
+    "type": "DatabricksSparkPython",
+    "typeProperties": {
+        "pythonFile": "dbfs:/entrypoint.py",
+        "libraries": [
+            {
+                "whl": "dbfs:/src-0.0.1-py3-none-any.whl"
+            }
+        ]
+    },
+    "linkedServiceName": {
+        "referenceName": "ADBLinkedServiceViaMSI",
```

Comment:
Instead of hard coding this twice (here and below in azurerm_data_factory_linked_service_azure_databricks), could you extract it into a local?

Reply:
Done.